ports/raspberrypi/bindings/rp2clock/OutputPin.c

```diff
@@ -0,0 +1,169 @@
+//|         pin: Pin to be used as clock input, allowed pins: 22,23,24,25
+//|         src: points to the source clock to be connected to the output pin.
+//|         divisor: Divisor for clock before it's driven onto pin.
+//|         """
+STATIC mp_obj_t rp2clock_outputpin_make_new(const mp_obj_type_t *type, size_t n_args,
+    size_t n_kw, const mp_obj_t *all_args) {
+    enum { ARG_pin, ARG_src, ARG_divisor };
+    static const mp_arg_t allowed_args[] = {
+        { MP_QSTR_pin,         MP_ARG_REQUIRED | MP_ARG_OBJ },
+        { MP_QSTR_src,         MP_ARG_OBJ | MP_ARG_KW_ONLY,  {.u_rom_obj = mp_const_none} },
+        { MP_QSTR_divisor,     MP_ARG_OBJ | MP_ARG_KW_ONLY,  {.u_obj = MP_OBJ_NEW_SMALL_INT(1)} },
+    };
+    mp_arg_val_t args[MP_ARRAY_SIZE(allowed_args)];
+    mp_arg_parse_all_kw_array(n_args, n_kw, all_args, MP_ARRAY_SIZE(allowed_args), allowed_args, args);
+
+    rp2clock_outputpin_obj_t *self = m_new_obj(rp2clock_outputpin_obj_t);
+    self->base.type = &rp2clock_outputpin_type;
+
+    // Validate pin number
+    common_hal_rp2clock_outputpin_validate_src_pin(args[ARG_pin].u_rom_obj);
+    self->pin = args[ARG_pin].u_rom_obj;
+    self->divisor = common_hal_rp2clock_outputpin_validate_divisor(mp_obj_get_float(args[ARG_divisor].u_obj));
+
+    // Validate pin based on clock
+    if (args[ARG_src].u_rom_obj != mp_const_none) {
+        self->src = validate_auxsrc(args[ARG_src].u_rom_obj, MP_QSTR_src);
+    } else {
+        self->src = AUXSRC_NONE;
+    }
+    self->enabled = false;
+    return MP_OBJ_FROM_PTR(self);
+}
+
+//|     def deinit(self) -> None:
+//|         """Releases the pin and frees any resources."""
+STATIC mp_obj_t rp2clock_outputpin_deinit(mp_obj_t self_in) {
+    rp2clock_outputpin_obj_t *self = MP_OBJ_TO_PTR(self_in);
+    // Release pin
+    common_hal_rp2clock_outputpin_deinit(self);
+    return mp_const_none;
+}
+STATIC MP_DEFINE_CONST_FUN_OBJ_1(rp2clock_outputpin_deinit_obj, rp2clock_outputpin_deinit);
+
+//|     def enable(self) -> None:
+//|         """Configures the pin and enables the clock output"""
+STATIC mp_obj_t rp2clock_outputpin_enable(size_t n_args, const mp_obj_t *pos_args, mp_map_t *kw_args) {
+    rp2clock_outputpin_obj_t *self = MP_OBJ_TO_PTR(pos_args[0]);
+    check_for_deinit(self);
+    common_hal_rp2clock_outputpin_enable(self);
+    return mp_const_none;
+}
+MP_DEFINE_CONST_FUN_OBJ_KW(rp2clock_outputpin_enable_obj, 1, rp2clock_outputpin_enable);
+
+//|     def disable(self) -> None:
+//|         """Disableds the pin and external clock"""
+STATIC mp_obj_t rp2clock_outputpin_disable(size_t n_args, const mp_obj_t *pos_args, mp_map_t *kw_args) {
+    rp2clock_outputpin_obj_t *self = MP_OBJ_TO_PTR(pos_args[0]);
+    check_for_deinit(self);
+    common_hal_rp2clock_outputpin_disable(self);
+    return mp_const_none;
+}
+MP_DEFINE_CONST_FUN_OBJ_KW(rp2clock_outputpin_disable_obj, 1, rp2clock_outputpin_disable);
+
+//|     src: rp2clock.AuxSrc
```

Comment:
Same thing here too. Can you just deinit and reconstruct instead of having enable, disable and .src?